### ISSUE TITLE
Replace URL hyperlinks with HyperlinkButton controls

### ISSUE THREAD

**Issue by donnie-msft** [NuGet/Home#10130]
Throughout PMUI, we should be using the built-in [HyperlinkButton ](https://docs.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.platformui.hyperlinkbutton?view=visualstudiosdk-2019) for any hyperlink which opens a URL.

Note: Any hyperlinks that don't actually open a URL should not be HyperlinkButtons. It's fine if they "look" like hyperlinks via a Style.

Example hyperlink to refactor:
https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageMetadataControl.xaml#L267
